subprojects/docs/src/docs/userguide/running-builds/performance.adoc

```diff
@@ -141,7 +141,7 @@ Daemons automatically shut down on memory pressure in Gradle 3.0 and above, so i
 
 [IMPORTANT]
 ====
-This feature is currently *_incubating_* and not enabled by default. This feature has the following limitations:
+This feature has the following limitations:
```

Comment:
I don't think saying "not enabled by default" here is necessary, as this whole section is about enabling CC.